test/endpoints/teams.accounts.spec.js

```diff
@@ -356,4 +357,62 @@ describe('Tests for /v1/teams/:teamId/accounts', () => {
       }
     );
   });
+  describe('GET: /v1/teams/:teamId/accounts', (done) => {
+    it(
+      'should not get a team account with invalid team ID',
+      (done) => {
+        const teamId = '7a2a5d4e-80b4-411a-8d79-0759b6ab06b0';
+        chai.request(server)
+          .get(`/v1/teams/${teamId}/accounts`)
+          .set('x-teams-user-token', mock.user0.token)
+          .end((err2, res) => {
+            res.should.have.status(200);
+            expect(res.body.errors)
+              .to.include('Team with the specified ID does not exist.');
+            expect(res.body.data).to.be.undefined;
+
+            done();
+          });
+      }
+    );
+    it('should get all team accounts', (done) => {
+      chai.request(server)
+        .post('/v1/teams')
+        .send(mock.team1)
+        .set('x-teams-user-token', mock.user0.token)
+        .end((err, res) => {
+          team1 = res.body.data.team;
+
+          chai.request(server)
+            .post(`/v1/teams/${team1.id}/accounts`)
+            .send(mock.account1)
+            .set('x-teams-user-token', mock.user0.token)
+            .end((err2, res2) => {
+              res2.should.have.status(200);
+              res2.body.should.have.property('data');
+              expect(res2.body.data).to.be.an('Object');
+              res2.body.data.should.have.property('account');
+              expect(res2.body.data.account.id).to.not.be.undefined;
+              expect(res2.body.data.account.teamId).to.equal(team1.id);
+              expect(res2.body.data.account.url).to.not.be.undefined;
+              expect(res2.body.errors).to.be.undefined;
+              chai.request(server)
+                .get(`/v1/teams/${team1.id}/accounts`)
+                .set('x-teams-user-token', mock.user0.token)
+                .end((err, res) => {
```

Comment:
Following the existing convention, these should be `err3` and `res3`:
```suggestion
                .end((err3, res3) => {
```

Reply:
This block is wrong because, apart from the very first one, none of your assertions even use `res` (which you should rename to `res3`). You didn't even bother to check the length of the array `res.body.data.accounts` to be sure `GET: /teams/:teamId/accounts` is working as expected. You pretty much just repeated the test assertions from above.